markdown/ch6.markdown

```diff
@@ -359,41 +371,104 @@ x = Dinv2 @ eigvecs[:, 1]
 ```
 
 (The reasons for using this vector are too long to explain here, but appear in
-the paper's supplementary material, linked above.)
+the paper's supplementary material, linked above. The short version is that
+choosing this vector minimizes the total length of the links between neurons.)
+
+There is one small kink that we must address before proceeding: eigenvectors
+are only defined up to a multiplicative constant. This follows simply from the
+definition of an eigenvector: suppose $v$ is an eigenvector of the matrix $M$,
+with corresponding eigenvalue $\lambda$. Then $\alpha v$ is also an eigenvector
+of $M$ for any scalar number $\alpha$, because
+$M(\alpha v) = \alpha Mv = \alpha \lambda v = \lambda (\alpha v)$. So, it is
+arbitrary whether a software package returns $v$ or $-v$ when asked for the
+eigenvectors of $M$. In order to make sure we reproduce the layout from the
+Varshney et al paper, we must make sure that the vector is pointing in the same
+direction, rather than the opposite direction. We do this by choosing an
+arbitrary neuron from their Figure 2, and checking the sign of `x` at that
+position.
```

Comment:
and then ...
I feel like there's a step missing here. Also it wasn't obvious how you are doing this in the code. Maybe add a couple of comments in the code for this?